tests/conftest.py

```diff
@@ -963,6 +963,64 @@ def test_run_data(bug_data):
     return {'test_runs': test_runs, 'push_time': time}
 
 
+@pytest.fixture
+def group_data(transactional_db, eleven_job_blobs, create_jobs):
+
+    query_string = '?manifest=/test&date=2022-10-01'
+
+    jt = []
+    jt.append(
+        th_models.JobType.objects.create(name="test-windows10-64-2004-qr/opt-mochitest-plain-1")
+    )
+    jt.append(
+        th_models.JobType.objects.create(name="test-windows10-64-2004-qr/opt-mochitest-plain-2")
+    )
+    jt.append(
+        th_models.JobType.objects.create(name="test-windows10-64-2004-qr/opt-mochitest-plain-swr-1")
+    )
+
+    g1 = th_models.Group.objects.create(name="/test")
+    for i in [0, 1, 2]:
+        job = eleven_job_blobs[i]
+        job['job'].update(
+            {
+                'taskcluster_task_id': 'V3SVuxO8TFy37En_6HcXL%s' % i,
+                'taskcluster_retry_id': '0',
+                'name': jt[i].name,
+            }
+        )
+        j = create_jobs([job])[0]
+
+        # when creating the job, we also create the joblog, we want the last job log entry
+        jls = th_models.JobLog.objects.all()
```

Comment:
This and the next line can be combined, the `last()` and `latest()` methods are [available](https://docs.djangoproject.com/en/4.1/ref/models/querysets/#last).